### PR TITLE
some changes, code reviews

### PR DESCRIPTION
**Pull request by Merloss**
life is good, birds are flying

### CODE REVIEW

File: static/js/presence.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-﻿// Define some usefull variables
```
Comment (eggsy) [inline static/js/presence.js:1]: `usefull` should be changed to `useful`

File: static/js/presence.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 // Initialize websocket session
 const lanyard = new WebSocket("wss://api.lanyard.rest/socket");
 
-var dscdata = {};
-var received = false;
+let dscdata = {};
+let received = false;
 
 // Subscribe for Discord ID
 lanyard.onopen = function () {
```
Comment (eggsy) [inline static/js/presence.js]: Use arrow function.

File: static/js/presence.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
-﻿// Define some usefull variables
-var statusIcon = document.getElementById("statusIcon");
-var listeningStatus = document.getElementById("listeningStatus");
-var listeningContent = document.getElementById("listeningContent");
+// Define some usefull variables
+let statusIcon = document.getElementById("statusIcon");
+let listeningStatus = document.getElementById("listeningStatus");
+let listeningContent = document.getElementById("listeningContent");
```
Comment (eggsy) [inline static/js/presence.js]: These were never reassigned later, can turn them into `const`'s.

File: static/js/theme.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 // HTML elements
-var main = document.getElementById("page")
-var switcher = document.getElementById("themeButton")
+let main = document.getElementById("page")
+let switcher = document.getElementById("themeButton")
```
Comment (eggsy) [inline static/js/theme.js]: These were never reassigned later, can turn them into `const`'s.

Comment (eggsy) [inline static/js/theme.js:3]: These were never reassigned later, can turn them into `const`'s.

File: static/js/theme.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
 // Settings
-var storedTheme = localStorage.theme;
+let storedTheme = localStorage.theme;
```
Comment (eggsy) [inline static/js/theme.js]: This never gets reassigned later, can be turned into a `const`.

File: static/js/theme.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 
 // Events
 switcher.addEventListener("click", themeEvent);
```
Comment (eggsy) [inline static/js/theme.js]: Use [`switcher.onclick`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onclick) instead.

File: static/js/theme.js [hunk 7]
```diff
@@ -37,6 +37,7 @@ function storeTheme(theme) {
         localStorage.theme = theme;
         return true
     } catch (e) {
+        console.log(e)
```
Comment (eggsy) [inline static/js/theme.js]: Use [`console.error`](https://developer.mozilla.org/en-US/docs/Web/API/console/error).

File: static/js/theme.js [hunk 8]
```diff
@@ -46,10 +47,8 @@ function changeTheme(theme) {
     if (theme == "dark") {
         // Enables the dark mode
         document.getElementById("page").classList.add("dark")
-    }
-
-    if (theme == "light") {
-        // Enables the dark mode
+    } else {
+        // Enables the light mode
         document.getElementById("page").classList.remove("dark")
```
Comment (eggsy) [inline static/js/theme.js]: This and the beginning of this `if` chain could be a single-liner.

**Review by eggsy — CHANGES_REQUESTED** [review]

**Comment by eri** [comment]
Thanks for your contribution, I'm gonna accept all these changes, however please note that this version of the website is almost a year old and I'm considering it as outdated, deprecated (or whatever you'd like to call it).

It will likely be gone on the next version of the website.

**Pull request merged**